common/Cargo.toml

```diff
@@ -32,5 +31,8 @@ siphasher = "0.3"
 unic-ucd-category = "0.9"
 volatile = "0.3"
 
+#[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = { version = "0.2.9", features = ["js"] }
```

Comment:
Is this intentional?

Reply:
This change was necessary to get the wasm build passing. It worked before because `RustPython` defines this feature in the `vm` crate and all features get merged.